### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by trev0r-STA**
All is in the title :-)

I've follow the explanation in [#3](https://github.com/jd1378/otphelper/pull/3)

If something should be modify, no problem.
And yes, two commits, but first time with github, and... It's hard

### CODE REVIEW

File: fastlane/metadata/android/fr/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Une application open-source qui vous permet de copier les OTP et les codes, des SMS et notifications, automatiquement en lisant toutes vos notifications.
+
+L'application fonctionne complètement hors ligne et sans permission internet. Vous pouvez donc être assuré que vos données ne quittent pas votre appareil.
+
+Si vous avez des problèmes ou des suggestions, soumettez-les dans notre dépôt GitHub :
+
+
+Traduit avec DeepL.com (version gratuite)https://github.com/jd1378/otphelper/issues
```
Comment (jd1378) [inline fastlane/metadata/android/fr/full_description.txt:8]: its okay if you use tools such as DeepL.com, but pay more attention to the output :)

**Comment by jd1378** [comment]
and the number of commits does not matter at all.

File: fastlane/metadata/android/fr/full_description.txt (hunk 1 above)
Comment (trev0r-STA) [reply fastlane/metadata/android/fr/full_description.txt]: Oh, f**k, I've take care of the meaning of translation, but don't see, the site add this ads.

**Comment by trev0r-STA** [comment]
Finally three commits 😁

**Comment by jd1378** [comment]
well nvm, I'll fix the rest
thanks for contributing :)

**Comment by trev0r-STA** [comment]
Revert the last commit. Not useful, finally
What means nvm ?
You're welcome

**Comment by jd1378** [comment]
It means dont mind it, as the test was failing constantly, it would waste your time 
I merged it to fix it locally :)

**Pull request merged**